<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>App</class>
 <widget class="QWidget" name="App">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Трекер финансов</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
  background-color: #f5f7fa; /* Светлый фон */
}

QPushButton {
  background-color: #4a90e2; /* Синий цвет */
  border: 1px solid #4a90e2;
  color: white;
  border-radius: 10px;
  padding: 5px 10px;
}

/* Наведение */
QPushButton:hover {
  background-color: #357ABD; /* Темно-синий при наведении */
}

/* Нажатие */
QPushButton:pressed {
  background-color: #2A65A0;
}

/* Отключенная кнопка */
QPushButton:disabled {
  background-color: #AFCDEB;
  border: 1px solid #AFCDEB;
  color: #ccc;
}

/* Улучшенный стиль QLabel */
QLabel {
  font-size: 16px; /* Увеличенный шрифт */
  font-weight: bold;
  color: #ffffff; /* Белый цвет текста */
  background-color: #4a90e2; /* Синий фон */
  padding: 6px;
  border-radius: 6px;
  text-align: center;
  font-family: &quot;Segoe UI&quot;, &quot;Arial&quot;, sans-serif; /* Интересный шрифт */
}

/* Стиль QLineEdit */
QLineEdit {
  border: 2px solid #ccc;
  border-radius: 8px;
  padding: 5px;
  background-color: #ecf0f1;
  color: black;
}

/* Улучшенный стиль для QDoubleSpinBox */
QDoubleSpinBox {
  background-color: #E3F2FD;
  border: 2px solid #90CAF9;
  border-radius: 8px;
  padding: 5px;
  color: #0D47A1;
  font-size: 14px;
}

/* Кнопки вверх/вниз */
QDoubleSpinBox::up-button, QDoubleSpinBox::down-button {
  background-color: #BBDEFB;
  border: 1px solid #90CAF9;
  width: 16px;
  border-radius: 4px;
}

/* Читаемые стрелки */
QDoubleSpinBox::up-arrow {
  image: url(:/icons/up-arrow.png); /* Заменить на путь к иконке */
  width: 10px;
  height: 10px;
}

QDoubleSpinBox::down-arrow {
  image: url(:/icons/down-arrow.png); /* Заменить на путь к иконке */
  width: 10px;
  height: 10px;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item>
    <widget class="QLabel" name="title">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>175</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>175</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Montserrat,sans-serif</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    color: #2c3e50;
    font-size: 50px;
    font-weight: bold;
    font-style: italic;
    font-family: 'Montserrat', sans-serif;
    text-shadow: 2px 2px 5px rgba(0, 0, 0, 0.3);
    padding: 10px 20px;
    background-color: #ecf0f1;
    border-radius: 8px;
    margin-bottom: 20px;
    display: block;
    text-align: center;
}
</string>
     </property>
     <property name="text">
      <string>Трекер финансов</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinAndMaxSize</enum>
     </property>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>75</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>75</height>
        </size>
       </property>
       <property name="text">
        <string>Прибыль за сегодня:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="today_profit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>75</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>75</height>
        </size>
       </property>
       <property name="text">
        <string>Потрачено за сегодня:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="today_spent">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="minimumSize">
        <size>
         <width>450</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Комментарий к пункту &quot;Потрачено за сегодня&quot;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="today_spent_commentary"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QPushButton" name="save_day_data_btn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Сохранить данные о доходе и расходе за сегодня</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="show_day_data_btn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Просмотреть сохранённые записи о доходе и расходе за сегодня</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="show_day_plot_btn">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>Показать сохранённые данные  о ежедневных доходах и расходах в виде графика</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinAndMaxSize</enum>
     </property>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>75</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>75</height>
        </size>
       </property>
       <property name="text">
        <string>Прибыль за неделю:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="week_profit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>75</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>75</height>
        </size>
       </property>
       <property name="text">
        <string>Потрачено за неделю:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="week_spent">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinAndMaxSize</enum>
     </property>
     <item>
      <widget class="QLabel" name="middle_week_income">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>75</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>75</height>
        </size>
       </property>
       <property name="text">
        <string>Средний доход за день составляет:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="income">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>75</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>75</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="count_income">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>Расчитать средний доход за день</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinAndMaxSize</enum>
     </property>
     <item>
      <widget class="QPushButton" name="save_week_data_btn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Сохранить данные о доходах и расходах за неделю</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="open_data">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Просмотреть сохранённые записи о доходах и расходах за неделю</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="show_plot_btn">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>Показать сохранённые данные о недельных доходах и расходах в виде графика</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
